src/internals/dispatcher.js

```diff
@@ -305,6 +305,14 @@ function useCallback<T>(callback: T, deps: Array<mixed> | void | null): T {
 
 function noop(): void {}
 
+function useTransition(): [() => void, boolean] {
+  return [noop, false]
+}
+
+function useDeferredValue<T>(input: T): T {
+  return input
```

Comment:
I think we’ll have to preserve the reference of `input` here to keep its reference equal?

Reply:
Do you mean across renders, so keep a `HookState` for it?